_includes/locale/Locale.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ */
+
+  declare(strict_types=1);
+
+  namespace Keyman\Site\com\keyman;
+
+  class Locale {
+    public const DEFAULT_LOCALE = 'en';
+
+    public const CROWDIN_LOCALES = array(
+      'en',
+      'es-ES',
+      'fr-FR'
+    );
```

Comment:
I think this should be in a separate file, or perhaps determined on a per-domain basis, because we have multiple l10n domains (i.e. keyboards is one localization domain). See my comments re filenames and folder names further down

Reply:
with the glob refactor below, this can go away